lib/main_common.pm

```diff
@@ -18,7 +18,7 @@ use testapi qw(check_var get_var get_required_var set_var check_var_array diag);
 use autotest;
 use utils;
 use version_utils qw(
-  is_hyperv_in_gui is_jeos is_gnome_next is_krypton_argon is_leap is_opensuse is_sle is_sles4sap is_sles4sap_standard leap_version_at_least sle_version_at_least is_desktop_installed is_installcheck is_rescuesystem is_staging is_tumbleweed
+  is_hyperv_in_gui is_jeos is_gnome_next is_krypton_argon is_leap is_opensuse is_sle is_sles4sap is_sles4sap_standard sle_version_at_least is_desktop_installed is_installcheck is_rescuesystem is_staging is_tumbleweed
```

Comment:
I really against such long function declaration lists , they don't give any benefits also there is high probability that they outdated so you can't actually trust them (e.g. lib/apachetest.pm). What if you just replace it with 
`use version_utils` ?

Reply:
ok I will do it in my PR :P

Reply:
This PR was not about making imports look pretty.